js/spa/test/test_chatinterface.reload.spec.ts

```diff
@@ -5,21 +5,20 @@ import { join } from "path";
 
 let _process;
 
-const demo_file = "chat_demo.py";
```

Comment:
I don't think we should change the name. There's another reload test that also uses "run.py" as the file name.